tests/page_objects/project_search_page.py

```
@@ -0,0 +1,60 @@
+            EC.element_to_be_clickable(ProjectSearchLocators.PROJECT_RESULT)
+        )
+        project_result.click()
+
+    def are_project_details_visible(self):
+        """Check if project details are visible."""
+        try:
+            self.wait.until(
+                EC.visibility_of_element_located(ProjectSearchLocators.PROJECT_DETAILS)
+            )
+            return True
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)